Rakefile

```diff
@@ -1,10 +1,6 @@
 require "bundler/gem_tasks"
-require "rake/testtask"
+require 'rspec/core/rake_task'
 
-Rake::TestTask.new(:test) do |t|
-  t.libs << "spec"
-  t.libs << "lib"
-  t.test_files = FileList["spec/**/*_spec.rb"]
-end
+RSpec::Core::RakeTask.new(:spec)
 
-task :default => :test
+task :default => :spec
```

Comment:
This means we can use the convention of running `rake` to trigger the tests, along with any other supporting tasks, such as linters.